editorconfig/versiontools.py

```diff
@@ -31,5 +33,4 @@ def split_version(version):
         split_version = list(match.groups())
         if split_version[3] is None:
             split_version[3] = "final"
-        split_version = list(map(int, split_version[:3])) + split_version[3:]
-        return tuple(split_version)
+        return (int(split_version[0]), int(split_version[1]), int(split_version[2]), split_version[3])
```

Comment:
the old code looks fine.

```suggestion
        return tuple(list(map(int, split_version[:3])) + split_version[3:])
```

Reply:
mypy doesn't understand the old code, that's why I changed it. (Also, I think it's much more readable after the change, but that's just my opinion.)